browser/src/map/handler/Map.Keyboard.js

```diff
@@ -734,6 +734,13 @@ L.Map.Keyboard = L.Handler.extend({
 			e.preventDefault();
 		}
 
+		// CTRL + ALT + N for new document. This needs to be handled by the integrator.
+		if (this._isCtrlKey(e) && e.altKey && e.key.toUpperCase() === 'N') {
+			this._map.fire('postMessage', {msgId: 'NewDocument', args: { type: this._map.getDocType() }});
```

Comment:
It doesn't use standard convention we have for other commands: COOL -> Host UI_xxxx (but there are already 2 exceptions... so not sure :) )

We have already command to create new document: UI_CreateFile
How this is different?
https://sdk.collaboraonline.com/docs/postmessage_api.html#id6

Reply:
Yep, thanks for the catch :) Replaced the command with the existing one.